lm_eval/tasks/jigsaw_unintended_bias.py

```diff
@@ -0,0 +1,145 @@
+# TODO: Remove all TODO comments once the implementation is complete.
+"""
+Jigsaw unintended bias in toxicity classification
+https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification
+Jigsaw Toxicity is a dataset curated by Alphabet from the now-defunct Civil Comments platform. It is used 
+to measure bias in toxicity classification models, specifically with equalized odds. In the equalized odds fairness notion, models
+that are less biased have smaller differences in False Positive Rate (FPR). Intuitively, this means that models do not 
+unjustly mislabel text with mentions of particular demographics as toxic. 
+Homepage: TODO: Add the URL to the task's Homepage here.
+"""
+import inspect
+import os
+import numpy as np
+import pandas as pd
+from lm_eval.base import PromptSourceTask
+import lm_eval.datasets.jigsaw_unintended_bias.jigsaw_unintended_bias
+from lm_eval.metrics import mean
+
+
+_CITATION = """
+https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification
+"""
+
+
+class JigsawUnintendedBias(PromptSourceTask):
+    """
+    Jigsaw Bias implementation. 
+    TODO: This implementation isn't correct for prompts that have >2 categorical outputs, since we 
+    need to further process false positive and true negatives for instances with non-binary categorical 
+    outputs. 
+    """
+    VERSION = 1
+    DATASET_PATH = "jigsaw_unintended_bias"
+    DATASET_NAME = None
+    dimension_to_identity_set = {"race": ["black", "white", "asian", "latino", "other_race_or_ethnicity"], 
+                                     "gender_nonbinary": ["male", "female", "transgender", "other_gender"]}
+
+    def __init__(self, **kwargs):
+        """
+        Jigsaw Toxicity must be downloaded from Kaggle from this url: 
+        https://www.kaggle.com/c/jigsaw-unintended-bias-in-toxicity-classification/data
+        """
+        data_dir = os.path.dirname(os.path.abspath(inspect.getfile(lm_eval.datasets.jigsaw_unintended_bias.jigsaw_unintended_bias)))
```

Comment:
Make `data_dir` a constructor argument with this path as the default. We want to allow users to specify jigsaw data locations as opposed to hardcoding the `lm_eval/datasets/jigsaw_unintended_bias` path.